esmvaltool/recipes/recipe_perfmetrics_CMIP5.yml

```diff
@@ -1897,7 +1897,61 @@ diagnostics:
 
   ### sic: SEA-ICE CONCENTRATION (SH) #########################################
 
-  ### dos: SOIL MOISTURE ######################################################
+  ### sm: SOIL MOISTURE #######################################################
+  sm:
+    description: Soil moisture
+    variables:
+      sm:
+        preprocessor: ppNOLEV1thr10
+        reference_dataset: ESACCI-SOILMOISTURE
+        mip: Lmon
+        field: T2Ms
+        derive: true
+        force_derivation: false
+        project: CMIP5
+        exp: historical
+        ensemble: r1i1p1
+        start_year: 2002
+        end_year: 2004
```

Comment:
This doesn't really matter for the implementation of sm but is there a reason why only 3 years are processed? ESACCI-SOILMOISTURE contains data for more than 30 years.

Reply:
All variables in this recipe cover only a few years for testing purposes until the recipe is complete.